packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.test.tsx

```diff
@@ -7,12 +7,25 @@ import { renderWithTheme } from 'src/utilities/testHelpers';
 import { CreateAlertDefinition } from './CreateAlertDefinition';
 describe('AlertDefinition Create', () => {
   it('should render input components', async () => {
-    const { getByLabelText } = renderWithTheme(<CreateAlertDefinition />);
+    const { getByLabelText, getByText } = renderWithTheme(
+      <CreateAlertDefinition />
+    );
 
+    expect(getByText('1. General Information')).toBeVisible();
     expect(getByLabelText('Name')).toBeVisible();
     expect(getByLabelText('Description (optional)')).toBeVisible();
     expect(getByLabelText('Severity')).toBeVisible();
+    expect(getByLabelText('Service')).toBeVisible();
+    expect(getByLabelText('Region')).toBeVisible();
+    expect(getByLabelText('Resources')).toBeVisible();
+    expect(getByText('2. Criteria')).toBeVisible();
+    expect(getByText('Metric Threshold')).toBeVisible();
+    expect(getByLabelText('Data Field')).toBeVisible();
+    expect(getByLabelText('Aggregation Type')).toBeVisible();
+    expect(getByLabelText('Operator'));
+    expect(getByLabelText('Threshold'));
```

Comment:
there is no assertion for operator and threshold

Reply:
Yeah , I will add them.